coderd/prometheusmetrics/prometheusmetrics_test.go

```diff
@@ -500,6 +500,41 @@ func TestAgentStats(t *testing.T) {
 	assert.EqualValues(t, golden, collected)
 }
 
+func TestExperimentsMetric(t *testing.T) {
+	t.Parallel()
+
+	log := slogtest.Make(t, nil).Leveled(slog.LevelDebug)
```

Comment:
I just copied this from elsewhere in the code; we don't need any level since the logger is not used yet, but if we add logging later we'll likely need to see debug logs in the test output.